api/src/main/java/org/apache/gravitino/SupportsCatalogs.java

```diff
@@ -98,6 +101,23 @@ Catalog createCatalog(
       Map<String, String> properties)
       throws NoSuchMetalakeException, CatalogAlreadyExistsException;
 
+  /**
+   * Create a managed catalog with specified catalog name, type, comment, and properties.
+   *
+   * @param catalogName the name of the catalog.
+   * @param type the type of the catalog.
+   * @param comment the comment of the catalog.
+   * @param properties the properties of the catalog.
+   * @return The created catalog.
+   * @throws NoSuchMetalakeException If the metalake does not exist.
+   * @throws CatalogAlreadyExistsException If the catalog already exists.
+   */
+  default Catalog createCatalog(
+      String catalogName, Catalog.Type type, String comment, Map<String, String> properties)
+      throws NoSuchMetalakeException, CatalogAlreadyExistsException {
+    return createCatalog(catalogName, type, null, comment, properties);
```

Comment:
```suggestion
      String name, Catalog.Type type, String comment, Map<String, String> properties)
      throws NoSuchMetalakeException, CatalogAlreadyExistsException {
    return createCatalog(name, type, null, comment, properties);
```

Reply:
This is out of scope of this PR as it's outside the CLI code or any changes made by this PR.  Best to raise a new issue for it.